### PR TITLE
Esp32c6 pm

### PR DESCRIPTION
**Pull request by FelipeMdeO**
*Note: Please adhere to [Contributing Guidelines](https://github.com/apache/nuttx/blob/master/CONTRIBUTING.md).*

## Summary

*Update this section with information on why change is necessary,
 what it exactly does and how, if new feature shows up, provide
 references (dependencies, similar problems and solutions), etc.*

## Impact

*Update this section, where applicable, on how change affects users,
 build process, hardware, documentation, security, compatibility, etc.*

## Testing

*Update this section with details on how did you verify the change,
 what Host was used for build (OS, CPU, compiler, ..), what Target was
 used for verification (arch, board:config, ..), etc. Providing build
 and runtime logs from before and after change is highly appreciated.*

### CODE REVIEW

**Comment by nuttxpr** [comment]
[**\[Experimental Bot, please feedback here\]**](https://github.com/search?q=repo%3Aapache%2Fnuttx+13552&type=issues)

__Fill In The Commit Message:__ This PR contains a Commit with an Empty Commit Message. Please fill in the Commit Message with the PR Summary.



This appears to be a template or placeholder for a NuttX Pull Request (PR), not an actual PR. Therefore, it cannot meet the NuttX Requirements because it contains no actual changes or information about a specific change. 

To assess if a PR meets the requirements, you need to provide the actual content of the PR, including:

* **A completed "Summary" section** explaining the reason for the change and its technical details.
* **A filled-out "Impact" section** outlining the consequences of the change on various aspects.
* **A comprehensive "Testing" section** detailing the verification process, including build environments, target platforms, and relevant logs demonstrating the change's effect. 

Once you provide a concrete PR with all these sections filled in, I can help you determine if it meets the NuttX Requirements.

File: boards/risc-v/esp32c6/common/scripts/esp32c6_sections.ld [hunk 1]
```diff
@@ -440,37 +440,105 @@ SECTIONS
...
+   */
   .rtc.data :
   {
     _rtc_data_start = ABSOLUTE(.);
-    *(.rtc.data)
-    *(.rtc.data.*)
-    *(.rtc.rodata)
-    *(.rtc.rodata.*)
+
+    mapping[rtc_data]
```
Comment (tmedicci) [inline boards/risc-v/esp32c6/common/scripts/esp32c6_sections.ld:493]: This syntax is not valid in NuttX (this is used on ESP-IDF, as you can check [here](https://docs.espressif.com/projects/esp-idf/en/stable/esp32/api-guides/linker-script-generation.html)). Can you double check this syntax and revert to the old (default for linker script) one?

File: boards/risc-v/esp32c6/common/scripts/esp32c6_sections.ld
Comment (tmedicci) [inline boards/risc-v/esp32c6/common/scripts/esp32c6_sections.ld:1]: Some tips about linker scripts: make sure it works for "default" configs. Does this work for `esp32c6-devkitc:nsh`, for instance?

Also, make sure your syntax work by checking the address assigned to some functions/sections by checking the `nuttx.map` file.

**Review by cederom — CHANGES_REQUESTED** [review]
Thank you @FelipeMdeO :-) Please update PR description and commit messages (topic + body) according to https://github.com/apache/nuttx/blob/master/CONTRIBUTING.md :-)

**Comment by FelipeMdeO** [comment]
Hello @cederom .

I open this Draft to get help from espressif guys. This is not ready and will take some time to be finished. This is my main activity for now and I hope finish as soon as possible.

**Pull request closed without merging**